atomic_reactor/plugins/pre_tag_from_config.py

```diff
@@ -91,7 +91,7 @@ def parse_and_add_tags(self):
 
     def add_registry_to_images(self):
         for image in self.workflow.tag_conf.images:
-            image.registry = next(iter(self.workflow.conf.registries))
+            image.registry = next(iter(self.workflow.conf.registry))
```

Comment:
I believe this should now be` self.workflow.conf.registry['uri']`

Reply:
updated